core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemActor.java

```diff
@@ -158,25 +166,15 @@ public void configure(FS fileSystem, ParameterList parameterList, INamedObject o
 			ConfigurationWarnings.add(owner, log, "action ["+ACTION_WRITE2+"] has been replaced with ["+ACTION_WRITE1+"]");
 			setAction(ACTION_WRITE1);
 		}
```

Comment:
Sorry to bother you again, but this part needs to be before the call to checkConfiguration(), e.g. at line 154, as it could modify the action.